SuperFlyTV/SuperConductor
# feat: update dependencies and tooling

This is a pretty large set of changes which tackles the following:

* update to yarn 4, and latest lerna
* update most dependencies
* update electron to 32 (current latest), and matching nodejs 20 for dev
* remove @sofie-automation/code-standard-preset. This is because it is a bit stuck with old versions of everything, in particular is uses and old version of typescript-eslint that doesnt work with typescript newer than 5.1 (5.2 was released ~august 2023).
* Convert everything to be ESM instead of commonjs. A few dependencies had issues with this (see the yarn patches), but most was fine, this allows for the next point
* Replace webpack with vite


My primary testing has been both dev and packaged builds on linux, and it appears to be happy in both flows.
The built binaries need some additional testing on windows and macos, in particular checking that the ci produced macos builds run on both intel and arm.

If anyone wants to review this, I would suggest skimming each commit in turn. In particular the ones which convert things to esm are large, but mostly just change imports
